AdvLoggerPkg/UnitTests/AdvancedLoggerWrapper/AdvancedLoggerWrapperTestApp.c

```diff
@@ -295,28 +298,27 @@ TestCursorWrappingMP (
   IN UNIT_TEST_CONTEXT  Context
   )
 {
-  BASIC_TEST_CONTEXT  *Btc;
-  EFI_STATUS          Status;
-  UINTN               Index;
-  UINTN               StrIndex;
-  UINTN               NumberOfProcessors;
-  UINTN               EnabledProcessors;
-  UINT8               *TempCache = NULL;
-  UNIT_TEST_STATUS    UtStatus;
-  UINTN               PrefixSize;
-  CHAR8               *EndPointer;
+  BASIC_TEST_CONTEXT         *Btc;
+  EFI_STATUS                 Status;
+  UINTN                      Index;
+  UINTN                      StrIndex;
+  UINTN                      NumberOfProcessors;
+  UINTN                      EnabledProcessors;
+  UINT8                      *TempCache = NULL;
+  UINTN                      PrefixSize;
+  CHAR8                      EndChar;
+  EFI_PROCESSOR_INFORMATION  CpuInfo;
 
   Btc = (BASIC_TEST_CONTEXT *)Context;
 
+  UT_ASSERT_TRUE (mLoggerInfo != NULL);

```

Comment:
small nit, for readability:
```suggestion
  UT_ASSERT_NOT_NULL (mLoggerInfo);

```